packages/contracts-bedrock/.storage-layout

```diff
@@ -135,8 +134,7 @@
 | xDomainMsgSender       | address                  | 204  | 0      | 20    | contracts/L2/L2CrossDomainMessenger.sol:L2CrossDomainMessenger |
 | msgNonce               | uint240                  | 205  | 0      | 30    | contracts/L2/L2CrossDomainMessenger.sol:L2CrossDomainMessenger |
 | failedMessages         | mapping(bytes32 => bool) | 206  | 0      | 32    | contracts/L2/L2CrossDomainMessenger.sol:L2CrossDomainMessenger |
-| reentrancyLocks        | mapping(bytes32 => bool) | 207  | 0      | 32    | contracts/L2/L2CrossDomainMessenger.sol:L2CrossDomainMessenger |
-| __gap                  | uint256[41]              | 208  | 0      | 1312  | contracts/L2/L2CrossDomainMessenger.sol:L2CrossDomainMessenger |
```

Comment:
I'm pretty sure this is fine, but would appreciate if @tynes can explicitly acknowledge it.

Reply:
Yes I agree that this is fine because the mapping was used transiently. The only way that a value would be left in there is if there is a bug in our code (static analysis using my eyes says no altho i didn't read the bytecode) or if ethereum is broken

Reply:
Resolving to continue reviews and get this across the line 🦾